src/injected/analyzers/tab-stops-orchestrator.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { WindowUtils } from 'common/window-utils';
+import { AllFrameRunnerTarget } from 'injected/all-frame-runner';
+import {
+    TabStopRequirementResult,
+    TabStopsRequirementEvaluator,
+} from 'injected/tab-stops-requirement-evaluator';
+import { TabbableElementGetter } from 'injected/tabbable-element-getter';
+import { FocusableElement } from 'tabbable';
+
+export class TabStopRequirementOrchestrator
+    implements AllFrameRunnerTarget<TabStopRequirementResult>
+{
+    public readonly name: string = 'TabStopRequirementOrchestrator';
+    private tabbableTabStops: FocusableElement[];
+    private actualTabStops: Set<HTMLElement> = new Set();
+    private latestVisitedTabStop: HTMLElement = null;
+    private reportResults: (payload: TabStopRequirementResult) => Promise<void>;
```

Comment:
It looks like we're not using `TabStopRequirementResults`, which contains the test id. Do we want to remove it and just add the id to TabStopRequirementResult?

Reply:
I've been going back and forth on how exactly we should propagate the results back ... I went with this because it's a bit simpler as is, doesn't require the array of results, and we do not intend to send objects back, at least in the orchestrator, multiples at a time.

Okay, after writing this out, I'm more convinced to include id on the result object itself; we can determine what to do with these result objects at a higher level than the orchestrator (maybe analyzer or some new layer between that).